### PR TITLE
Update the list of authors

### PR DESCRIPTION
**Pull request by agiudiceandrea**
Goal: update the list of "Authors" in https://docs.qgis.org/testing/en/docs/user_manual/preamble/contributors.html.
https://github.com/qgis/QGIS-Documentation/graphs/contributors?from=2016-10-31&to=2022-06-26&type=c

The committers without a real name in GitHub: @muranamihdk, @MorriganR.

Ticket(s): #

- [x] Backport to LTR documentation is requested

### CODE REVIEW

**Comment by gacarrillor** [comment]
Thanks @agiudiceandrea, mine is Germán Carrillo, just in case you want to update it. If not, I can create a PR for that.

**Comment by thayeray** [comment]
I’ve only contributed once. But my full name is Thayer Young. 

    On Sunday, June 26, 2022, 12:12:48 PM EDT, Andrea Giudiceandrea ***@***.***> wrote:  
 
 
Goal: update the list of "Authors" in https://docs.qgis.org/testing/en/docs/user_manual/preamble/contributors.html.
https://github.com/qgis/QGIS-Documentation/graphs/contributors?from=2016-10-31&to=2022-06-26&type=c

The committers without a real name in GitHub: @lajunek, @felixfeckler, @roya0045, @jveenstra1, @muranamihdk, @ttomasz, @pathmapper, @MorriganR, @thayeray, @gacarrillor, @rldhont.

Ticket(s): #
   
   -  Backport to LTR documentation is requested

You can view, comment on, or merge this pull request online at:

  https://github.com/qgis/QGIS-Documentation/pull/7655

Commit Summary
   
   - cefecd6 Update the list of authors

File Changes 
 
(1 file)
   
   -  M docs/user_manual/preamble/contributors.rst (12) 

Patch Links:
   
   - https://github.com/qgis/QGIS-Documentation/pull/7655.patch
   - https://github.com/qgis/QGIS-Documentation/pull/7655.diff

File: docs/user_manual/preamble/contributors.rst [hunk 1]
```diff
@@ -44,13 +44,21 @@ and update the whole QGIS documentation.
...
-   "Zoltan Siki", "Håvard Tveite", "Mattheo Ghetta", "Salvatore Larosa", "Konstantinos Nikolaou"
+   "Zoltan Siki", "Håvard Tveite", "Matteo Ghetta", "Salvatore Larosa", "Konstantinos Nikolaou"
    "Tom Chadwin", "Larry Shaffer", "Nathan Woodrow", "Martina Savarese", "Godofredo Contreras"
    "Astrid Emde", "Luigi Pirelli", "Thomas Gratier", "Giovanni Allegri", "GiordanoPezzola"
    "Paolo Corti", "Tudor Bărăscu", "Maning Sambale",  "Claudia A. Engel", "Yoichi Kayama"
    "Otto Dassau", "Denis Rouzaud", "Nick Bearman", "embelding", "ajazepk"
    "Ramon", "Andrei", "zstadler",  "icephale", "Rosa Aguilar"
-   "Patrice Pineault"
+   "Patrice Pineault", "lajunek", "felixfeckler", "Benoît de Mezzo", "Étienne Trimaille"
+   "Andrea Giudiceandrea", "Julien Cabieces", "roya0045", "Sebastian Gutwein", "jveenstra1"
```
Comment (DelazJ) [inline docs/user_manual/preamble/contributors.rst:54]: not sure of Alex full name

**Review by DelazJ — COMMENTED** [review]
for those i know...

File: docs/user_manual/preamble/contributors.rst (hunk 1 above)
Comment (roya0045) [reply docs/user_manual/preamble/contributors.rst:54]: My github id is fine.

**Comment by agiudiceandrea** [comment]
> for those i know...

Thanks @DelazJ!

File: docs/user_manual/preamble/contributors.rst [hunk 2]
```diff
@@ -44,13 +44,20 @@ and update the whole QGIS documentation.
...
    "Tom Chadwin", "Larry Shaffer", "Nathan Woodrow", "Martina Savarese", "Godofredo Contreras"
    "Astrid Emde", "Luigi Pirelli", "Thomas Gratier", "Giovanni Allegri", "GiordanoPezzola"
    "Paolo Corti", "Tudor Bărăscu", "Maning Sambale",  "Claudia A. Engel", "Yoichi Kayama"
    "Otto Dassau", "Denis Rouzaud", "Nick Bearman", "embelding", "ajazepk"
    "Ramon", "Andrei", "zstadler",  "icephale", "Rosa Aguilar"
-   "Patrice Pineault"
+   "Patrice Pineault", "Jörn Gutzeit", "Felix Feckler", "Benoît de Mezzo", "Étienne Trimaille"
+   "Andrea Giudiceandrea", "Julien Cabieces", "roya0045", "Sebastian Gutwein", "jveenstra1"
+   "Ryan Welfle", "Martin Pergler", "Ivan Ivanov", "muranamihdk", "Loïc Bartoletti"
+   "ttomasz", "Ian Maddaus", "Jürnjakob Dugge", "Roman Bug", "Damiano Lombardi"
```
Comment (ttomasz) [inline docs/user_manual/preamble/contributors.rst:56]: My name is Tomasz Taraś

Comment (agiudiceandrea) [reply docs/user_manual/preamble/contributors.rst:56]:
```suggestion
   "Tomasz Taraś", "Ian Maddaus", "Jürnjakob Dugge", "Roman Bug", "Damiano Lombardi"
```

File: docs/user_manual/preamble/contributors.rst [hunk 3]
```diff
@@ -44,13 +44,20 @@ and update the whole QGIS documentation.
...
-   "Zoltan Siki", "Håvard Tveite", "Mattheo Ghetta", "Salvatore Larosa", "Konstantinos Nikolaou"
+   "Zoltan Siki", "Håvard Tveite", "Matteo Ghetta", "Salvatore Larosa", "Konstantinos Nikolaou"
    "Tom Chadwin", "Larry Shaffer", "Nathan Woodrow", "Martina Savarese", "Godofredo Contreras"
    "Astrid Emde", "Luigi Pirelli", "Thomas Gratier", "Giovanni Allegri", "GiordanoPezzola"
    "Paolo Corti", "Tudor Bărăscu", "Maning Sambale",  "Claudia A. Engel", "Yoichi Kayama"
    "Otto Dassau", "Denis Rouzaud", "Nick Bearman", "embelding", "ajazepk"
    "Ramon", "Andrei", "zstadler",  "icephale", "Rosa Aguilar"
-   "Patrice Pineault"
+   "Patrice Pineault", "Jörn Gutzeit", "Felix Feckler", "Benoît de Mezzo", "Étienne Trimaille"
+   "Andrea Giudiceandrea", "Julien Cabieces", "roya0045", "Sebastian Gutwein", "jveenstra1"
```
Comment (agiudiceandrea) [inline docs/user_manual/preamble/contributors.rst:54]:
```suggestion
   "Andrea Giudiceandrea", "Julien Cabieces", "roya0045", "Sebastian Gutwein", "Jessica Veenstra"
```

**Pull request merged**